src/core_plugins/elasticsearch/lib/create_admin_cluster.js

```diff
@@ -0,0 +1,19 @@
+export default function (server) {
+  const config = server.config();
+  const Logger = server.plugins.elasticsearch.ElasticsearchClientLogging;
+
+  class AdminClientLogging extends Logger {
```

Comment:
This pulls its parent class from the server object, which is passed into the function `server.plugins.elasticsearch.ElasticsearchClientLogging`.  /cc @tylersmalley thoughts on importing this instead?